docs/src/_contribute/ui.md

```diff
@@ -1,126 +0,0 @@
----
-title: UI Development
-description: Meltano is open source software built by a growing team and a community of contributors.
-layout: doc
-weight: 10
-hidden: true
----
-
-## UI Development
-
-In the event you are contributing to Meltano UI and want to work with all of the frontend tooling (i.e., hot module reloading, etc.), you will need to run the following commands:
-
-```bash
-# Activate your poetry created virtual environment if needed.
-# If you manage your virtualenv activation through other means you can omit this command.
-poetry shell
-
-# Create a new Meltano project
-meltano init $PROJECT_DIRECTORY
-
-# Change directory into your newly created project
-cd $PROJECT_DIRECTORY
-
-# Start the Meltano API and a production build of Meltano UI that you can ignore
-meltano ui
-
-# Open a new terminal tab and go to the directory you cloned meltano into
-cd $WHEREVER_YOU_CLONED_MELTANO
-
-# Install frontend infrastructure at the root directory
-yarn setup
-
-# Start local development environment
-yarn serve
-```
-
-The development build of the Meltano UI will now be available at <http://localhost:8080/>.
-
-A production build of the API will be available at <http://localhost:5000/> to support the UI, but you will not need to interact with this directly. However, as mentioned in the [API Development section](/contribute/api) above, users on MacOS may need to specify an alternate [bind to port](/reference/settings#ui-bind-port) to prevent a port conflict with a MacOS system service also running on port 5000.
-
-
-<div class="notification is-warning">
-  <p><strong>If you're developing for the _Embed_ app (embeddable <code>iframe</code> for reports) you can toggle <code>MELTANO_EMBED</code>:</strong></p>
-
-<pre>
-# Develop for the embed app
-export MELTANO_EMBED=1
-
-# Develop for the main app (this is the default)
-export MELTANO_EMBED=0
-
-# Start local development environment
-yarn serve
-</pre>
-</div>
```

Comment:
Difficult to do a thorough comparison but I didn't see this contributing content (`yarn serve`, etc.) and could be valuable to preserve in reference page:

- https://deploy-preview-6955--meltano.netlify.app/reference/ui